datanode/entities/entities.go

```diff
@@ -15,7 +15,7 @@ package entities
 import "google.golang.org/protobuf/proto"
 
 type PagedEntity[T proto.Message] interface {
-	Market | Party | Trade | Order | MarginLevels | MarketData | Reward | Candle | Deposit | Withdrawal | Asset | OracleSpec | OracleData | Position | LiquidityProvision | Vote | Transfer | AccountBalance | Proposal | Delegation | Node | NetworkParameter | Checkpoint | StakeLinking
+	Market | Party | Trade | Order | MarginLevels | MarketData | Reward | Candle | Deposit | Withdrawal | Asset | OracleSpec | OracleData | Position | LiquidityProvision | Vote | Transfer | AccountBalance | Proposal | Delegation | Node | NetworkParameter | Checkpoint | StakeLinking | NodeSignature | KeyRotation | ERC20MultiSigSignerAddedEvent | ERC20MultiSigSignerRemovedEvent
```

Comment:
**question (non-blocking)**:  I can see entities in here that are not reflected in the GraphQL schema. Could you check to see that all of them are? Specifically `ERC20MultiSigSignerAddedEvent` jumped out.

Reply:
There are a few functions that are exposed on the gRPC level that haven't made it through to the GraphQL side of things. So far in this migration, we have only migrated existing functionality of the GraphQL and gRPC functions.

We did have a discussion yesterday about this yesterday, it shouldn't take much work to plumb in the gRPC functions to the GraphQL API, but for now I've not added anything "new" to the GraphQL API, I've only deprecated functions that needed to be replaced with the newer cursor based paginated function.

I hope that makes sense.

Reply:
It makes sense not to add new functionality, but we have recently had a few bugs that come about due to fields in the GRPC not being added/mapped/reflected properly in the GraphQL schema, resulting in the graphql server throwing errors (https://github.com/vegaprotocol/data-node/issues/594 for a recent example)

I guess I'm just asking for a sanity check that if these new fields are in GRPC, and I was subscribed in GraphQL to `busEvents`, and one of these was emitted, that it wouldn't break the subscription by emitting a server side error?

Reply:
Absolutely, there are some other plumbing that's missing in the GraphQL resolvers which I'm currently working through. I'll make sure everything is mapped correctly.